lib/active_merchant/billing/gateways/visanet_peru.rb

```diff
@@ -77,7 +77,7 @@ def scrub(transcript)
         transcript.
           gsub(%r((Authorization: Basic )\w+), '\1[FILTERED]').
           gsub(%r((\"cardNumber\\\":\\\")\d+), '\1[FILTERED]').
-          gsub(%r((\"cvv2Code\\\":)\d+), '\1[FILTERED]')
+          gsub(%r((\"cvv2Code\\\":\\\")\d+), '\1[FILTERED]')
```

Comment:
@duff my bad for not committing the test credentials. I wasn't sure if it's okay to commit them to the repo. A quick check with @rwdaigle and I can add them.